src/client/entity-editor/common/language-field.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2016  Ben Ockmore
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import {Input} from 'react-bootstrap';
+
+import React from 'react';
+import VirtualizedSelect from 'react-virtualized-select';
+
+function LanguageField({
+	...props
+}) {
+	return (
+		<Input
+			label="Language"
+		>
+			<VirtualizedSelect {...props}/>
```

Comment:
It means more boilerplate, but how would you feel about explicitly specifying these props? It makes it a little difficult at this level to see what behavior will be affected at the `VirtualizedSelect` level.